codepropertygraph/src/main/resources/schemas/base.json

```diff
@@ -255,7 +255,8 @@
         {"id" : 3, "name" : "AST", "comment" : "Syntax child" , "keys" : [] },
         {"id" : 19, "name" : "CFG", "comment" : "Control flow", "keys" : [] },
 
-        {"id" : 9, "name" : "CONTAINS", "keys" : ["LOCAL_NAME", "INDEX"], "comment" : "Shortcut to contained node"},
+        {"id" : 9, "name" : "CONTAINS_NODE", "keys" : ["LOCAL_NAME", "INDEX"], "comment" : "Edge from container to contained node"},
+        {"id" : 28, "name" : "CONTAINS", "keys" : ["LOCAL_NAME", "INDEX"], "comment" : "Shortcut to contained node"},
```

Comment:
I am pretty sure the keys have just been introduced as the already existing CONTAINS edge was hijacked for something else. So please reconfirm and than remove those here and on internal branch.

Reply:
Hmm, for some reason I thought there was a difference in cardinality, but can't see where that came from. Detangling...